src/psyclone/tests/psyir/transformations/intrinsics/intrinsic2code_trans_test.py

```diff
@@ -32,36 +32,36 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 # Author: R. W. Ford, STFC Daresbury Laboratory
-# Modified: A. R. Porter, STFC Daresbury Laboratory
+# Modified: A. R. Porter and S. Siso, STFC Daresbury Laboratory
 
-'''Module containing tests for the operator abstract class which
-provides common functionality for the intrinsic operator
-transformations (such as MIN, ABS and SIGN).'''
+'''Module containing tests for the Intrinsic2CodeTrans abstract class which
+provides common functionality for the intrinsic transformations (such as MIN,
+ABS and SIGN).'''
 
-from __future__ import absolute_import
 import pytest
 from psyclone.psyir.transformations import TransformationError
```

Comment:
pylint says that UnaryOperation is unused.

Reply:
Removed